src/char/i_char.py

```diff
@@ -34,11 +35,13 @@ def __init__(self, skill_hotkeys: Dict, screen: Screen, template_finder: Templat
     def _discover_capabilities(self) -> CharacterCapabilities:
```

Comment:
It looks good. But I’m not an expert on how color  filters work. However the main challenge was to distinguish between 22 charges and 2 charges for instance that’s why >95 was used for original low_on_charges tracking. I think there is potentially an issue too